install_config/persistent_storage/pod_security_context.adoc

```diff
@@ -742,7 +742,8 @@ label.
 This means two things for unprivileged containers:
 
 - The volume will be given a `type` which is accessible by unprivileged
-containers. This `type` is usually *svirt_sandbox_file_t*.
+containers. Previously, the label specified was `svirt_sandbox_file_t`. This label
```

Comment:
In never versions of the policy, it's called `container_file_t`, which might be worth mentioning as well.